src/signals_notebook/entities/chemical_drawing.py

```diff
@@ -16,6 +17,46 @@
 log = logging.getLogger(__name__)
 
 
+class ChemicalDrawingPosition(str, Enum):
+    REACTANTS = 'reactants'
+    REAGENTS = 'reagents'
+    PRODUCTS = 'products'
+
+
+class ChemicalStructure(str, Enum):
+    REACTANT = 'reactant'
+    REAGENT = 'reagent'
+    PRODUCT = 'product'
+
+
+class ChemicalStructureFormat(str, Enum):
+    INCHI = 'inchi'
+    CDXML = 'cdxml'
+
+
+class Structure(BaseModel):
+    id: str
+    type: ChemicalStructure
+    inchi: Optional[str]
+    cdxml: Optional[str]
```

Comment:
Why do we need both of properties at the same time? We use only one of them. It would be better to have one body property and format

Reply:
WE can have situatuon that one of them is None and response returns both of them. It can be helpfull to have both of them